### ISSUE TITLE
Shellcheck useless-suppression à la pylint

### ISSUE THREAD

**Issue by LLyaudet** [koalaman/shellcheck#3407]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows a potential problem:

```sh
#!/bin/bash

# shellcheck disable=SC1000
a="WAT"

echo "${a}"
```

#### Here's what shellcheck currently says:
No warning at all.

#### Here's what I wanted to see:
A warning for SC9999, useless-suppression like in pylint:
```
Warning SC9999:  useless-suppression
The suppression for SC1000 at line 3 is useless:
# shellcheck disable=SC1000
No violation of SC1000 is reported if you remove it.
```

**Comment by LLyaudet** [comment]
Hello @e-kwsm :),

Thanks for closing this as duplicate and sorry about the duplicate.
At least it may help other people wanting the same feature find something similar by full-text search before creating another duplicate.
Once all the sensible way of titling the issue have been found approximately.

Best regards,
    Laurent Lyaudet
